tests/test_review_fixes.py

```
@@ -0,0 +1,139 @@
+@pytest.fixture
+def temp_xml(tmp_path):
+    def _create_xml(body_xml: str) -> Path:
+        xml = f'<?xml version="1.0" encoding="utf-8"?><w:document {NS}><w:body>{body_xml}</w:body></w:document>'
+        xml_path = tmp_path / "test_doc.xml"
+        xml_path.write_text(xml)
+        return xml_path
+
+    return _create_xml
+
+
+def _make_manager(xml_path):
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `_make_manager`

(ANN202)